src/main/java/org/springframework/data/jpa/repository/query/QueryUtils.java

```diff
@@ -201,10 +201,6 @@ public static String getExistsQueryString(String entityName, String countQueryPl
 
 	/**
 	 * Returns the query string for the given class name.
-	 *
```

Comment:
Why remove the JavaDoc? Shouldn't we rather document the nullability here? Same for removals below.

Reply:
Originally removed empty JavaDoc attributes to minimize noise.
Put those back in and added comments about nullability + annotations.